Components/Coroutines/Town/TrinityTownRun.cs

```diff
@@ -35,11 +39,13 @@ public static bool IsVendoring
             set => s_vendorProperty.Value.SetValue(null, value);
         }
 
-
         public static bool HasMaterialsRequired => Core.Inventory.Currency.HasCurrency(Zeta.Game.TransmuteRecipe.UpgradeRareItem);
+
         public static bool IsRareToLegendaryTransformationPossible(List<ItemSelectionType> types = null)
         {
-            if (!ZetaDia.IsInGame || !ZetaDia.IsInTown) return false;
+            if (!ZetaDia.IsInGame ||
+                !ZetaDia.IsInTown)
+                return false;

```

Comment:
Add curly brackets when the statement is multi-line. Although the statement can be a one liner here